Palace.xcodeproj/project.pbxproj

```diff
@@ -3936,7 +3936,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 56;
+				CURRENT_PROJECT_VERSION = 57;
```

Comment:
Could you also change project version to 1.0.19 please? 1.0.18 has already been published.